scripts/scil_assign_color_to_trk.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import load_tractogram, save_tractogram
+import numpy as np
+
+from scilpy.io.utils import (assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_overwrite_arg)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description='Assign an hexadecimal RGB color to a Trackvis TRK '
+                    'tractogram. The hexadecimal RGB color should be '
+                    'formatted as 0xRRGGBB',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    p.add_argument('in_tractogram',
+                   help='Tractogram.')
+
+    p.add_argument('out_tractogram',
+                   help='Colored TRK tractogram.')
+    p.add_argument('color',
+                   help='Can be either hexadecimal (ie. #RRGGBB or 0xRRGGBB).')
```

Comment:
Doesn't work properly with # format. The parser don't take the argument. We must to write the color as "#RRGGBB".